backend/btrixcloud/profiles.py

```diff
@@ -405,25 +388,14 @@ async def get_profile_name(
 
         return ""
 
-    async def get_crawl_configs_for_profile(
-        self, profileid: UUID, org: Organization
-    ) -> list[CrawlConfigProfileOut]:
-        """Get list of crawl configs with basic info for that use a particular profile"""
-
-        crawlconfig_info = await self.crawlconfigs.get_crawl_config_info_for_profile(
-            profileid, org
-        )
-
-        return crawlconfig_info
-
     async def delete_profile(
         self, profileid: UUID, org: Organization
     ) -> dict[str, Any]:
         """delete profile, if not used in active crawlconfig"""
-        profile = await self.get_profile_with_configs(profileid, org)
+        profile = await self.get_profile(profileid, org)
 
-        if len(profile.crawlconfigs) > 0:
-            return {"error": "in_use", "crawlconfigs": profile.crawlconfigs}
+        if profile.inUse:
+            return {"error": "in_use"}
```

Comment:
This is leading to an InternalServiceError, because the output doesn't match the ResponseModel attached to this endpoint. Seems like it might have been a bug that's been around for a minute, but could use some reworking. The frontend is also expecting the whole Profile object to be returned in the response, which is also not the case.